File: sujet_machine/exemples/recette/skeleton/recette.m
#import <Foundation/Foundation.h>
#include<stdio.h>
#include<stdlib.h>


void recette0(int n, int* ordre, int* desordre) {
    /* Inserez votre code ici */
}

int main(void){
  NSAutoreleasePool * pool = [[NSAutoreleasePool alloc] init];
  int bb, ba, n;
  scanf("%d ", &n);
  int *ordre = calloc(n, sizeof(int));
  for (ba = 0; ba < n; ba++)
      scanf("%d ", &ordre[ba]);
  int *desordre = calloc(n, sizeof(int));
  for (bb = 0; bb < n; bb++)
      scanf("%d ", &desordre[bb]);
  recette0(n, ordre, desordre);
  [pool drain];
  return 0;
}

File: sujet_machine/exemples/recette/skeleton/recette.ml
let recette0 _n _ordre _desordre =
  (* Inserez votre code ici *)
  ()

let () =
 let n = Scanf.scanf "%d " (fun n -> n) in
  let ordre = Array.init n (fun _bc ->
    let bd = Scanf.scanf "%d " (fun bd -> bd) in
    bd) in
  let desordre = Array.init n (fun _be ->
    let bf = Scanf.scanf "%d " (fun bf -> bf) in
    bf) in
  recette0 n ordre desordre 
 